src/app/app.component.ts

```diff
@@ -1,10 +1,24 @@
 import { Component } from '@angular/core';
+import { ApiserviceService } from 'src/app/service/apiservice.service';
+import { Vendor } from 'src/app/types/types';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
 })
 export class AppComponent {
-  title = 'TFR-Management';
+  vendors: Vendor[] = [];
+
+   title = 'TFR-Management';
+  constructor(private service: ApiserviceService) {}
+  ngOnInit() {
```

Comment:
This does not need here. the component who is using this  vendor component may do the implementation